src/patterns/task-list-pages/index.md.njk

```diff
@@ -51,7 +53,9 @@ Where possible, task names should:
 
 ### Show the status of the tasks
 
-Include a summary above the task list to say how many tasks or sections have been completed.
+Include a summary above the task list to say how many tasks have been completed.
+
+You might find a better word to describe what a ‘task’ is for your users. In an application form, each task might a ‘section’ of questions to complete.
 
 ![A screenshot showing a task list summary that says 'Application incomplete. You have completed 3 of 8 sections'](task-list-summary.png)
```

Comment:
To match house style, can this screenshot be replaced to make 'you have' a contraction 'you've'?
```suggestion
![A screenshot showing a task list summary that says 'Application incomplete. You've completed 3 of 8 sections'](task-list-summary.png)
```

Reply:
```suggestion
![A screenshot showing a task list summary that says 'Application incomplete. You’ve completed 3 of 8 sections'](task-list-summary.png)
```